packages/plugin/src/generators/create-package/create-package.ts

```diff
@@ -149,57 +152,29 @@ async function createCliPackage(
  * @param options
  * @returns
  */
-async function addE2eProject(host: Tree, options: NormalizedSchema) {
-  const pluginProjectConfiguration = readProjectConfiguration(
-    host,
-    options.project
-  );
-  const pluginOutputPath =
-    pluginProjectConfiguration.targets.build.options.outputPath;
-
-  const cliProjectConfiguration = readProjectConfiguration(
-    host,
-    options.projectName
-  );
-  const cliOutputPath =
-    cliProjectConfiguration.targets.build.options.outputPath;
-
-  const e2eTask = await e2eProjectGenerator(host, {
-    pluginName: options.projectName,
-    projectDirectory: options.projectDirectory,
-    pluginOutputPath,
-    npmPackageName: options.name,
-    skipFormat: true,
-    rootProject: false,
-  });
+function addE2eProject(host: Tree, options: NormalizedSchema) {
+  const projects = getProjects(host);
+  const isRootProject = projects.get(options.project).root === '.';
+  const e2eProjectName = isRootProject ? 'e2e' : `${options.project}-e2e`;
+  if (!projects.has(e2eProjectName)) {
+    return;
+  }
```

Comment:
Take in the e2e project to add the test to.

Reply:
changed